config/config.go

```diff
@@ -317,6 +321,8 @@ type GlobalConfig struct {
 	OpsGenieAPIHost  string `yaml:"opsgenie_api_host" json:"opsgenie_api_host"`
 	VictorOpsAPIURL  string `yaml:"victorops_api_url" json:"victorops_api_url"`
 
+	SMTPTLSConfig *config.TLSConfig `yaml:"smtp_tls_config"`
```

Comment:
I don't think a struct like this works as a default, all the rest are simple types. This should possibly be per-receiver only.